app/src/main/java/com/teamagam/gimelgimel/app/mainActivity/view/MainActivityPanel.java

```diff
@@ -157,6 +169,14 @@ public void onPanelStateChanged(View panel, SlidingUpPanelLayout.PanelState prev
                                         SlidingUpPanelLayout.PanelState newState) {
             sLogger.userInteraction("MainActivity's bottom panel mode changed from "
                     + previousState + " to " + newState);
+            if (isOpenedState(newState)) {
```

Comment:
Add here the condition of mOnPanelOpenListener != null and save the need of the NO_OP_LISTENER and the handling of it.